src/Http/Routing/src/RouteValuesAddress.cs

```diff
@@ -24,4 +25,7 @@ public class RouteValuesAddress
     /// Gets or sets ambient route values from the current HTTP request.
     /// </summary>
     public RouteValueDictionary? AmbientValues { get; set; }
+
+    /// <inheritdoc />
+    public override string? ToString() => $"{RouteName}({string.Join(',', ExplicitValues.Select(kv => $"{kv.Key}=[{kv.Value}]"))})";
```

Comment:
If we end up doing this, we should cache it.

Reply:
Why do you think we should cache it?